src/com/googlecode/jmxtrans/model/output/CloudWatchWriter.java

```diff
@@ -0,0 +1,154 @@
+package com.googlecode.jmxtrans.model.output;
+
+import com.amazonaws.auth.InstanceProfileCredentialsProvider;
+import com.amazonaws.regions.Region;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.cloudwatch.AmazonCloudWatchClient;
+import com.amazonaws.services.cloudwatch.model.MetricDatum;
+import com.amazonaws.services.cloudwatch.model.PutMetricDataRequest;
+import com.googlecode.jmxtrans.model.Query;
+import com.googlecode.jmxtrans.model.Result;
+import com.googlecode.jmxtrans.util.BaseOutputWriter;
+import com.googlecode.jmxtrans.util.ValidationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Writes data to <a href="http://aws.amazon.com/cloudwatch/">AWS CloudWatch</a> using the AWS Java SDK
+ *
+ * @author Sascha Moellering <moelleri@amazon.de>
+ */
+
+public class CloudWatchWriter extends BaseOutputWriter {
+
+    /** Logger. */
+    private static final Logger log = LoggerFactory.getLogger(CloudWatchWriter.class);
+
+    private AmazonCloudWatchClient cloudWatchClient;
+    private String namespace;
+
+    public static final String AWS_NAMESPACE = "namespace";
+    public static final String METADATA_URL = "http://169.254.169.254/latest/dynamic/instance-identity/document";
+    public static final String REGION = "region";
+
+    public void doWrite(Query query) throws Exception {
+        PutMetricDataRequest metricDataRequest = new PutMetricDataRequest();
```

Comment:
Indentation should be done with tabs, not spaces to be coherent with other sources in this project.